lib/connection-manager.js

```diff
@@ -88,7 +87,8 @@ const checkHealth = (config) => {
 // Initiate Connection
 const connectLogServer = (config) => {
     firstrun = true;
-    debug(config.LOGDNA_LOGURL);
+    config.LOGDNA_LOGURL = `${config.LOGDNA_LOGSSL ? 'https' : 'http'}://${config.LOGDNA_LOGHOST}:${config.LOGDNA_LOGPORT}`;
```

Comment:
Same comment as above. You don't need (and shouldn't) modify the `config` object. Save `logdnaURL` as a local `const`.